### ISSUE TITLE
Dependency differences from KERN-2

### ISSUE THREAD

**Issue by ameenjakoet** [idia-astro/idia-container-kern-3#1]
The packages below have not included in the current KERN-3 container build. Some of these are dependencies of each other (i.e. mostly the CUDA packages). Also, we should keep track of the packages being added to the KERN-3 suite [here](https://github.com/kernsuite/packaging/issues):

- purify
- mt-imager

- [x] python-montblanc
- [x] python-pycuda
- [x] nvidia-cuda-toolkit
- [x] python-pycuda-doc
- [x] python-mako
- [x] nvidia-cuda-dev 
- [x] nvidia-profiler 
- [x] libcuinj64-7.5 
- [x] libcuda-5.5-1 (replaced with libcuda1-361; rather that 5.5-1 is a virtual package of 361)

- libpurify-dev
- libpurify2.0
- libawimager2lib
- liblofarft
- libpythondppp
- aips

**Comment by ameenjakoet** [comment]
Found this issue on the KERN suite repo: [Issue #71](https://github.com/kernsuite/packaging/issues/71)
Links are provided on the site of their progress. Seems mt-imager is not going to be included, purify depends on old packages, and aips is supposed to be updated.

**Comment by foxmouldy** [comment]
I think that this repo needs to be the vanilla K3 build that will work on things like the "Fat Nodes" and the SLURM/HPC nodes. If you're okay with this, then I propose the following:

- Don't worry about AIPS. AIPS usage is highly specialized and I can't think of any users right now. If there is a strong/urgent motivation/request, then we can build this in another container.
- Right now the use case for the vanilla K3 container has a weak dependence on GPU supported modules for the moment. So you can leave them out as well (i.e., CUDA based stuff).
- We don't need PURIFY/mt-imager, so leave them out. 
- Montblanc - its useful to have this, but not essential. So leave this build for the end. We can always incorporate it into the next version of the container.

I hope that solves a few problems.

**Comment by arusstaylor** [comment]
I agree about AIPS. I dont anticipate any use of this.

I think the helo source finding group wanted a container for GPU to run PINK.    Is that a separate container?

Russ

**Comment by foxmouldy** [comment]
Yes, there is a separate container for Source Finding/Pink. I think that its best not to mix use-cases.

**Comment by ameenjakoet** [comment]
@foxmouldy sorted out dependency issues with Montblanc - managed to complete installation and support in sandbox mode.
